nautobot_ssot/integrations/servicenow/urls.py

```diff
@@ -0,0 +1,8 @@
+"""URL patterns for nautobot-ssot-servicenow."""
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("servicenow/config/", views.SSOTServiceNowConfigView.as_view(), name="servicenow_config"),
```

Comment:
How is this view accessed by the user? Is there a menu item link for it, or a link from the SSOT dashboard?

Reply:
I was not able to test this as I do not have access to any ServiceNow instance.

Reply:
@chadell Any chance to test this PR?

Reply:
Shouldn't need live access to a ServiceNow instance in order to test this.